### ISSUE TITLE
A option to sync timestamp from NewPipe to other devices and continue with MPV or VLC

### ISSUE THREAD

**Issue by ztpnk** [TeamNewPipe/NewPipe#3248]
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I checked if the issue/feature exists in the latest version.

I think this wouldn't be used really much but it would be great if there is an option to export/sync the saved timestamps from NewPipe to sync it with PC's and other devices and continue with vlc or mpv for example at the exact same moment I stopped on my phone without to remember it every time.

Im not an Android/Java Developer but I could write an script for mpv or an browser extension together with the "server backend" to sync it.

Its similiar to #1163 but maybe its possible to seperate it from the other cloud features since the app already saves the timestamp of videos... otherwise sorry for creating an "duplicate".

**Comment by gkeegan** [comment]
Good idea, but I don't know if it would be possible to have NewPipe write to the watch history of VLC.

**Comment by ztpnk** [comment]
> 
> 
> Good idea, but I don't know if it would be possible to have NewPipe write to the watch history of VLC.

Thats why it needs a server backend or something. A option in NewPipe where the data is written to the server in json for example and a plugin-script for VLC which read the data from the server.

**Comment by grravity** [comment]
I'd like to see this with Invidious login sync
